### PR TITLE
Feat/style profile page

### PR DESCRIPTION
**Pull request by yasminegha**

### CODE REVIEW

File: frontend/src/styles/ProfilePage.css [hunk 1]
```diff
@@ -48,14 +53,19 @@
 
 .header-username {
     font-size: 45px;
-    font-weight: 100;
-    margin: 0;
+    font-weight: 200;
+    margin: 2;
```
Comment (ZacharyBys) [inline frontend/src/styles/ProfilePage.css]: You're defining margin as well as margin left here, it's redundant. Also add units to margin

File: frontend/src/styles/ProfilePage.css [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+    padding-bottom: 0px;
+    margin-bottom:8px;
+    background-color: rgba(196, 196, 196, 0.31);
+    border-radius: 2px;
+    width: 1000px;
 }
 
 .profile-page-content {
     flex-grow: 3;
     padding: 0 1em;
```
Comment (ZacharyBys) [inline frontend/src/styles/ProfilePage.css:14]: Units

File: frontend/src/styles/ProfilePage.css [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
-    background: white;
+    background: #202020;
 }
-
-.profile-page-container li {
-    color: black;
+.caption-container , .caption-container-accepted , .caption-container-rejected {
+    padding-bottom: 0px;
+    margin-bottom:8px;
+    background-color: rgba(196, 196, 196, 0.31);
```
Comment (ZacharyBys) [inline frontend/src/styles/ProfilePage.css]: Be consistent with your color notation

**Review by ZacharyBys — DISMISSED** [review]
Small changes

**Review by tehblasian — APPROVED** [review]
Everything looks good. I just made sure everything was properly merged with dev, and I adjusted some CSS.

**Review by ZacharyBys — APPROVED** [review]

**Pull request merged**